External/CSG/Classes/CSG_Model.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using UnityEngine;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace UnityEngine.ProBuilder.Csg
+{
+    /// <summary>
+    /// Representation of a mesh in CSG terms. Contains methods for translating to and from UnityEngine.Mesh.
+    /// </summary>
+    sealed class CSG_Model
+    {
+        internal List<CSG_Vertex> vertices;
```

Comment:
Is there a reason why this is a property instead of being a backing field with proper accessor? It feels inconsistent with `m_Materials` and `m_Indices` as backing fields.

Reply:
I just didn't bother to change it from what the library had in the first place